### ISSUE TITLE
String.strip/1 is deprecated. Use String.trim/1 instead

### ISSUE THREAD

**Issue by basicobject** [devinus/poison#184]
Elixir version: 1.7.3
Poison version 3.1

**Comment by kumarn33** [comment]
I can work on this if no one is already working on this.

**Comment by VSPPedro** [comment]
Hi, @kumarn33

Any new on this issue?

It would be good to solve this too:

```
warning: Integer.to_char_list/2 is deprecated. Use Integer.to_charlist/2 instead
  lib/poison/encoder.ex:173
```

**Comment by cburgmer** [comment]
See #185

**Comment by devinus** [comment]
Fixed in 5.0.
